libraries/botbuilder-dialogs/src/prompts/confirmPrompt.ts

```diff
@@ -130,4 +126,12 @@ export class ConfirmPrompt extends Prompt<boolean> {
 
         return result;
     }
+
+    private DetermineCulture(activity: Activity) : string {
```

Comment:
The method name should be camel cased, and there is an extra space after `)`

Reply:
Could this method be used elsewhere (e.g. ChoicePrompt), if the object we're examining is passed in as an argument?

```ts
public determineCulture(activity: Activity, defaultChoiceOptions: { [culture: string]: any }): any {
```

Or is this too sloppy?

Reply:
easier to add as private and make public later...